.travis.yml

```diff
@@ -65,6 +68,9 @@ before_deploy:
 - |
   # Stage 3, Step 2: Deploy the earthhub
   python ./deploy.py --build --push --deploy earthhub
+- |
+  # Stage 3, Step 2: Deploy the earthhub
```

Comment:
can you indulge the pedantic side of me and name it `# Stage 3, Step 3: Deploy the bootcamp-hub`? Or delete the comment. Incorrect comments make me nervous :)